sympy/parsing/c/c_parser.py

```diff
@@ -256,6 +256,16 @@ def transform_var_decl(self, node):
                     ).as_Declaration(
                         value = val
                     )
+                elif (child.kind == cin.CursorKind.CHARACTER_LITERAL
```

Comment:
Can you add a comment here describing when it will be called?

Reply:
Sure, will add it in the next commit.